plugins/mongodb.yaml

```diff
@@ -113,7 +113,7 @@ pipeline:
     severity:
       parse_from: severity
       mapping:
-        critical: F
+        fatal: F
```

Comment:
`critical` should map to `error2`

Reply:
This one I'm going to disagree with. The F in this circumstance stands for Fatal, so I think this is a more accurate translation of the severity. Our original mapping was not as accurate.

Reply:
Fair enough, I've been reviewing this on a "will this have parity" basis. This'll show up differently in the UI, and IDK if that'll effect any dashboards or anything.